rdbms/src/test/java/io/spine/server/storage/jdbc/JdbcAggregateStorageShould.java

```diff
@@ -79,6 +87,38 @@ public void throw_exception_when_closing_twice() throws Exception {
         storage.close();
     }
 
+    @Test
+    public void return_history_iterator_with_specified_batch_size() throws SQLException {
+        final JdbcAggregateStorage<ProjectId> storage = getStorage(ProjectId.class,
+                                                                   TestAggregateWithMessageId.class);
+        final int batchSize = 10;
+        final AggregateReadRequest<ProjectId> request = new AggregateReadRequest<>(newId(),
+                                                                                   batchSize);
+        final DbIterator<AggregateEventRecord> iterator =
+                (DbIterator<AggregateEventRecord>) storage.historyBackward(request);
+
+        // Use `PreparedStatement.getFetchSize()` instead of `ResultSet.getFetchSize()`,
+        // because the result of the last one depends on a JDBC driver implementation.
```

Comment:
`...because the result of the latter depends on ...`

Reply:
Done.